index.d.ts

```diff
@@ -0,0 +1,25 @@
+interface CodeReturn<T> {
```

Comment:
`CodeReturn` is rather literal, how about `ImageDataLike<T>` or something?

Reply:
Much better name, I like it